src/wp-includes/blocks/navigation.php

```diff
@@ -1358,7 +1358,7 @@ function block_core_navigation_get_most_recently_published_navigation() {
  * @param WP_Post $post         `wp_navigation` post object corresponding to the block.
  * @return string Serialized inner blocks in mock Navigation block wrapper, with hooked blocks inserted, if any.
  */
-function block_core_navigation_insert_hooked_blocks( $inner_blocks, $post = null ) {
+function block_core_navigation_insert_hooked_blocks( $inner_blocks, $post ) {
```

Comment:
Would making `$post` now to be required could cause `Fatal error: Uncaught ArgumentCountError: Too few arguments to function` if someone is using it without it? Was this function introduced before 6.5?

Reply:
Looks like this has been discussed. See https://github.com/WordPress/gutenberg/pull/58379#issuecomment-1914671732 cc @ockham